vtr_flow/scripts/python_libs/vtr/flow.py

```diff
@@ -119,7 +119,11 @@ def run(
 
         check_incremental_sta_consistency :
             Do a second-run of the incremental analysis to compare the result files
-
+        
+        check_route:
+            Check first placement by enabling VPR analysis.
```

Comment:
Can you change this to "Check first placement and routing by enabling VPR analysis".